build/tools/avalanche-bridge-faq.md

```diff
@@ -0,0 +1,139 @@
+
+### Are there tutorials on how to use the bridge?
+ Yes, you can view video tutorials for all bridge functionality [here](https://www.youtube.com/playlist?list=PLRHl-ulWK4-FPRA7SS1OrCOC8cOc2K8sP).
+
+### Can I send to a different wallet address?
+ After crossing the bridge in either direction, you can send assets to any wallet or contract address you would like to. However, the bridge was designed to only allow transfers across networks from and to the same wallet address. 
+ 
+### Can I safely speed up my transaction?
+ Yes, you can click on the “Speed Up” button on Metamask. However, keep in mind of [this FAQ](https://docs.avax.network/build/tools/avalanche-bridge-faq#what-happens-after-I-have-sped-up-a-transaction-on-metamask). 
+
+### What happens after I have sped up a transaction on Metamask?
+ “Speeding up” a transaction through metamask issues a new transaction on Ethereum that has a higher gas price than the transaction that was originally sent. Since the new transaction has a high gas price, it is more likely to be included in a block, and since both transactions use the same nonce, only one will be accepted into the blockchain. Speeding up a transaction that is transferring funds to the bridge is perfectly safe; however, the UI is not currently aware of the new transaction, meaning you may not see the confirmations animation. Once the new transaction has 35 confirmations on Ethereum, check your metamask wallet on Avalanche to see the wrapped funds.
+
+### When transferring from Avalanche to Ethereum, why does the number showing on Metamask not match the number I requested?
+ When transferring from Avalanche to Ethereum, 0 shows up on Metamask, not the actual number of tokens to be transferred. This is a known problem with Metamask.
+
+## Fees
+### How do fees work on the Avalanche Bridge?
+ The bridge charges certain transfer fees in order to cover the cost of the transaction fees on each blockchain network. These fees are charged in kind of the ERC20 asset being transferred. Currently, the fees are calculated as follows.
+
+For moving assets from Ethereum to Avalanche, the fee is a fixed small percentage of the assets being transferred, capped at a reasonable amount to incentivize larger transfers. However, as a welcome gesture to the Avalanche community, the fee percentage when transferring from Ethereum to Avalanche is currently 0.00%. Further, transfers to Avalanche may qualify for an AVAX airdrop as explained below in [this FAQ](https://docs.avax.network/build/tools/avalanche-bridge-faq#How-does-the-airdrop-work).
```

Comment:
Can 'a fixed small percentage' be replaced with an actual number, as well as the capped number?

Reply:
@michaelkaplan13 @learyce can you help?

Reply:
No because it's based on the price of the assets so will be constantly fluxiating

Reply:
Removed the `fixed` in the sentence based on feedback from Conor.

Reply:
IMHO, this is a pretty unsatisfactory answer and will only generate more questions on support channels. We should strive to provide more clarity is at all possible. 'Small percentage' and 'reasonable amount' are pretty vague, if it is based on the price of the asset, why can't we say that and provide some examples?

Reply:
Let's update to "a small portion of the assets transferred equal to the estimated gas costs of moving the funds out of the Ethereum wallet plus a small buffer (currently set to $5)"

Reply:
Updated.

Reply:
Yep, much clearer.